Graphviz Error

```
Exception in thread "RenderGraphvizTree" java.lang.NoClassDefFoundError: org/apache/batik/util/XMLConstants
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at org.apache.batik.bridge.SVGSVGElementBridge.createGraphicsNode(SVGSVGElementBridge.java:141)
	at org.apache.batik.bridge.GVTBuilder.build(GVTBuilder.java:76)
	at org.apache.batik.transcoder.SVGAbstractTranscoder.transcode(SVGAbstractTranscoder.java:208)
	at org.apache.batik.transcoder.image.ImageTranscoder.transcode(ImageTranscoder.java:92)
	at org.apache.batik.transcoder.XMLAbstractTranscoder.transcode(XMLAbstractTranscoder.java:142)
	at org.apache.batik.transcoder.SVGAbstractTranscoder.transcode(SVGAbstractTranscoder.java:156)
	at guru.nidi.graphviz.engine.BatikRasterizer.doRasterize(BatikRasterizer.java:53)
	at guru.nidi.graphviz.engine.SvgRasterizer.rasterize(SvgRasterizer.java:31)
	at guru.nidi.graphviz.engine.BatikRasterizer.rasterize(BatikRasterizer.java:27)
	at guru.nidi.graphviz.engine.Renderer.toImage(Renderer.java:74)
	at guru.nidi.graphviz.engine.Renderer.toFile(Renderer.java:51)
	at us.ihmc.avatar.drcRobot.RobotDefinitionTreeRenderer.<init>(RobotDefinitionTreeRenderer.java:314)
	at us.ihmc.nadia.rdx.apps.NadiaRDXModelViewer$1.lambda$renderImGuiWidgets$2(NadiaRDXModelViewer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.ClassNotFoundException: org.apache.batik.util.XMLConstants
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 23 more
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -41,7 +41,7 @@ testDependencies {
 graphvizDependencies {
    api(ihmc.sourceSetProject("main"))
 
-   api("guru.nidi:graphviz-java:0.5.4")
+   api("guru.nidi:graphviz-java-all-j2v8:0.18.1")
 }
 
 yovariablesDependencies {

